packages/pn-pa-webapp/src/api/apiKeys/__test__/ApiKeys.api.test.ts

```diff
@@ -23,33 +20,51 @@ describe('Api keys api tests', () => {
   mockAuthentication();
 
   it('getApiKeys', async () => {
-    const mock = new MockAdapter(apiClient);
-    mock.onGet(APIKEY_LIST()).reply(200, mockApiKeysFromBE);
+    mock = mockApi(
+      apiClient,
+      'GET',
+      APIKEY_LIST(),
+      200,
+      undefined,
+      mockApiKeysFromBE
+    );
     const res = await ApiKeysApi.getApiKeys();
     expect(res).toStrictEqual(mockApiKeysFromBE);
+    expect(mock.history.get).toHaveLength(1);
```

Comment:
here this expect and the next one is useless. you can remove it here and for all tests of the api